### PR TITLE
Add new resources to be shared by automated and manual athena queries

### PR DESCRIPTION
**Pull request by andylangridge**
* Added some new workgroups and a glue database to sit alongside the existing data analysis infra - these are currently not in use
* We will share the Athena Output bucket between manual and automated queries going forward - separation by path in the bucket
* Added SSM params for new resources
* Ordered SSM params alphabetically as I was having trouble finding things!

### CODE REVIEW

File: data-analysis/template.yaml [hunk 1]
```diff
@@ -137,6 +159,34 @@ Resources:
...
+      Description: Workgroup for manual queries on TxMA audit data
+      Name: manual-audit-data-queries
+      RecursiveDeleteOption: true
+      WorkGroupConfiguration:
+        EnforceWorkGroupConfiguration: true
+        PublishCloudWatchMetricsEnabled: false
+        ResultConfiguration:
+          EncryptionConfiguration:
+            EncryptionOption: SSE_S3
+          OutputLocation: !Sub s3://${AthenaQueryOutputBucket}/manual-audit-data-queries
```
Comment (gds-topickering) [inline data-analysis/template.yaml:174]: I think the output locations might need a trailing '/'

File: data-analysis/template.yaml [hunk 2]
```diff
@@ -137,6 +159,34 @@ Resources:
...
+      Description: Workgroup for automated queries on TxMA audit data by TICF/Zendesk integration
+      Name: ticf-automated-audit--data-queries
+      RecursiveDeleteOption: true
+      WorkGroupConfiguration:
+        EnforceWorkGroupConfiguration: true
+        PublishCloudWatchMetricsEnabled: false
+        ResultConfiguration:
+          EncryptionConfiguration:
+            EncryptionOption: SSE_S3
+          OutputLocation: !Sub s3://${AthenaQueryOutputBucket}/ticf-automated-audit-data-queries
```
Comment (gds-topickering) [inline data-analysis/template.yaml:188]: Same point re trailing '/'

File: data-analysis/template.yaml (hunk 1 above)
Comment (andylangridge) [reply data-analysis/template.yaml:174]: Fixed!

File: data-analysis/template.yaml (hunk 2 above)
Comment (andylangridge) [reply data-analysis/template.yaml:188]: Fixed!

**Review by mabon-gds — APPROVED** [review]

**Pull request merged**